src/kOS.Safe/Compilation/KS/KSScript.cs

```diff
@@ -19,46 +19,38 @@ public KSScript()
 
         public override List<CodePart> Compile(string filePath, int startLineNum, string scriptText, string contextId, CompilerOptions options)
         {
-            List<CodePart> parts = null;
```

Comment:
you are right that the null check was not germane to your change. i figured why while we are in here we can clean it up :)